scripts/tests/forest_cli_check.sh

```diff
@@ -55,6 +55,12 @@ pushd "$(mktemp --directory)"
     if "$FOREST_CLI_PATH" snapshot validate --check-network mainnet "$validate_me"; then
         exit 1
     fi
+
+    : : check that it contains at least one expected checkpoint
+    # If calibnet is reset or the checkpoint interval is changed, this check has to be updated
+    if "$FOREST_CLI_PATH" archive checkpoints "$validate_me" | grep bafy2bzaceatx7tlwdhez6vyias5qlhaxa54vjftigbuqzfsmdqduc6jdiclzc; then
```

Comment:
I'm not sure we even need the `if` condition, given the `-e` flag to the script. It's not a big deal, though.